.storybook/webpack.config.js

```diff
@@ -23,9 +23,8 @@ const sassFunctions = require('bpk-mixins/sass-functions');
 
 const postCssPlugins = require('../scripts/webpack/postCssPlugins');
 
-const { BPK_TOKENS, ENABLE_CSS_MODULES } = process.env;
```

Comment:
What was the purpose of ENABLE_CSS_MODULES var? Also wondering if we use BPK_TOKENS as well

Reply:
It looks to be super old configuration which going through our docs and setup we never actually use anywhere so looks to be an irrelevant variable.

It also doesn't line up and work with the configuration we need to provide in the newer `css-loader` so that we can specify our `localIndentName` as it would either be a boolean or an object, which we always need to provide the `localIndentName`, `modules` would always need to be an object :) 

Hope that helps, let me know if not